danielyxie/bitburner
# Bugfix/2486 auto complete for connect and backdoored server

closes #2486

It is possible to connect to backdoored servers, but the auto complete for connect does not work for them. This is fixed with this PR. I have also updated the connect help text.
Tested with:
- run backdoor on "zer0"
- connect to home
- type `connect z` + tab
=> autocompleted to `connect zer0`

- run `help connect`
=> Additional info is shown, that you can also connect to backdoored servers
